flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlDdlToOperationConverterTest.java

```diff
@@ -908,17 +908,15 @@ public void testCreateTableWithFullDataTypes() {
                         createTestItem(
                                 "MAP<BIGINT, BOOLEAN>",
                                 DataTypes.MAP(DataTypes.BIGINT(), DataTypes.BOOLEAN())),
-                        // Expect to be ROW<`f0` INT NOT NULL, `f1` BOOLEAN>.
                         createTestItem(
                                 "ROW<f0 INT NOT NULL, f1 BOOLEAN>",
                                 DataTypes.ROW(
-                                        DataTypes.FIELD("f0", DataTypes.INT()),
+                                        DataTypes.FIELD("f0", DataTypes.INT().notNull()),
```

Comment:
This hits the previous fix in https://github.com/apache/flink/blob/c226ea938adbf1c397a3045b0f80fc9e2c623148/flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/calcite/FlinkTypeFactory.scala#L433